extensions/applicationinsights-properties-js/Tests/Unit/src/properties.tests.ts

```diff
@@ -36,6 +36,7 @@ export class PropertiesTests extends AITestClass {
         this.addUserTests();
         this.addDeviceTests();
         this.addTelemetryTraceTests();
+        this.addSessionTests();
```

Comment:
Note: If this PR is related to an issue, please use the issue name and reference the issue number as it makes the release notes a LOT easier to create and for people to follow.

If not, please raise an Issue and reference it as well.

Reply:
Got it, I updated the description, is that correct?

Reply:
And the title